template-parts/single.php

```diff
@@ -15,6 +15,9 @@
 		<?php the_content(); ?>
 	</div>
 
+	<div class="page-comments">
```

Comment:
It's already wrapped at https://github.com/pojome/elementor-hello-theme/blob/master/comments.php#L11
(and not exist if comments are not supported)